conf/README.md

```diff
@@ -178,3 +185,9 @@ Now set up the systemd service:
 sudo systemctl enable /opt/webwork/webwork2/conf/webwork2.service
 sudo systemctl start webwork2
 ```
+
+### Notes for setting up nginx with SSL
+
+If you are running SSL on nginx, follow the instructions for your certificate management for
+doing this.  If you use Let's encrypt to give certificates, this is a helpful
+[blog post](https://www.nginx.com/blog/using-free-ssltls-certificates-from-lets-encrypt-with-nginx/).
```

Comment:
I don't think this should be added.  Setting up ssl for production is beyond the scope of what I think should be done in this file.  That is for the actual installation instructions on the webwork wiki.  In addition, LetsEncrypt is not what should be promoted for true production usage.

Furthermore, this is inconsistent with the apache2 deployment instructions as no comment is made there about how to specifically set up ssl.

Really, this isn't even specifically a webwork2 set up step.  It is a more general nginx server set up step.

Reply:
True.  I've started a set of install notes on the wiki and will add that there.